### PR TITLE
let maintainers know that three shall be the number

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: conda_forge_tick/migrators.xsh [hunk 1]
```diff
@@ -413,7 +413,12 @@ class Version(Migrator):
             "- [ ] Dependencies have been updated if changed\n"
             "- [ ] Tests have passed \n"
             "- [ ] Updated license if changed and `license_file` is packaged \n"
-            "\n")
+            "\n"
+            "Note that the bot will stop issuing PRs if more than 3 Version bump PRs"
+            "generated by the bot are open. If you don't want to package a particular"
```
Comment (scopatz) [inline conda_forge_tick/migrators.xsh]: Single spaces needed at the ends of these lines inside the quotes

File: conda_forge_tick/migrators.xsh [hunk 2]
```diff
@@ -413,7 +413,12 @@ class Version(Migrator):
             "- [ ] Dependencies have been updated if changed\n"
             "- [ ] Tests have passed \n"
             "- [ ] Updated license if changed and `license_file` is packaged \n"
-            "\n")
+            "\n"
+            "Note that the bot will stop issuing PRs if more than 3 Version bump PRs"
```
Comment (mariusvniekerk) [inline conda_forge_tick/migrators.xsh]:
```suggestion
            "Note that the bot will stop issuing PRs if more than 3 Version bump PRs "
```

File: conda_forge_tick/migrators.xsh (hunk 1 above)
Comment (mariusvniekerk) [inline conda_forge_tick/migrators.xsh]:
```suggestion
            "generated by the bot are open. If you don't want to package a particular "
```

File: conda_forge_tick/migrators.xsh (hunk 2 above)
Comment (mariusvniekerk) [inline conda_forge_tick/migrators.xsh]: We should pull the number 3 from a parameter and not hardcode it

**Comment by CJ-Wright** [comment]
@regro/auto-tick ready for another round

**Pull request merged**